packages/cozy-client/examples/web/src/Example.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react'
+import { useQuery, Q } from 'cozy-client'
+
+const styles = {
+  data: {
+    maxHeight: 200,
+    overflow: 'scroll',
+    background: 'lightyellow',
+    padding: '1rem'
+  },
+
+  stateTester: {
+    background: 'lavender',
+    padding: '1rem'
+  },
+
+  queries: {
+    display: 'flex'
+  },
+
+  block: {
+    border: '3px dotted',
+    margin: '1rem',
+    width: '50%',
+    padding: '1rem'
+  },
+
+  amount: {
+    positive: {
+      color: 'teal'
+    },
+    negative: {
+      color: 'crimson'
+    }
+  }
+}
+
+const accountsConn = {
+  as: 'accounts',
+  query: () => Q('io.cozy.bank.accounts')
+}
+
+const transactionsConn = {
+  as: 'transactions',
+  query: () => Q('io.cozy.bank.operations').limitBy(10).sortBy([{'date': 'desc'}]).where({})
+}
+
+const QueryResult = ({ collection }) => {
+
+  return (
+    <div style={styles.data}>
+      id: { collection.id }<br/>
+      bookmark: {collection.bookmark }<br/>
+      length: {collection.data ? collection.data.length : 0}{' '}
+    </div>
+  )
+}
+
+const useCounter = initialCount => {
+  const [count, setCount] = useState(initialCount)
+  const increment = () => setCount(count + 1)
+  const decrement = () => setCount(count - 1)
+  return [count, increment, decrement]
+}
+
+const ColoredAmount = ({ amount }) => {
+  return <span style={amount > 0 ? styles.amount.positive : styles.amount.negative}>{ amount }</span>
+}
+
+const TransactionRow = ({ transaction: tr }) => (
+  <div>
+    {tr.date.slice(0, 10)} - {tr.label} - <ColoredAmount amount={tr.amount} />
+  </div>
+)
+
+const Queries = () => {
+  const [count, increment, decrement] = useCounter(0)
+  const accounts = useQuery({ query: accountsConn })
```

Comment:
The API is `useQuery({ query: { query: myQueryDef, as: 'name' }})` I find the double `query` a bit strange, though I have no problem with `useQuery({ query: { definition: myQueryDef, as: 'name' }})`. Is one officially preferred over the other? Do we need to keep both around?

Reply:
Simplified in https://github.com/cozy/cozy-client/commit/c7ee67e4383c244e9d527b8f01da6f7877909e20